modules/api/src/test/java/com/intuit/wasabi/api/AssignmentsResourceTest.java

```diff
@@ -203,6 +208,6 @@ public void postBatchAssignmentForPage() throws Exception {
 
     @Test
     public void getAssignmentsQueueLength() throws Exception {
-        assertNotNull(resource.getAssignmentsQueueLength());
+        assertTrue(resource.getAssignmentsQueueLength().getStatus() == HttpStatus.SC_OK);
```

Comment:
a hamcrest equivalent would be, psuedocode:

  assertThat(resource.getAssignmentQueueLength(), is(SC_OK))

what say you?